src/Sulu/Bundle/AdminBundle/Resources/js/containers/Link/overlays/ExternalLinkTypeOverlay.js

```diff
@@ -170,8 +170,17 @@ class ExternalLinkTypeOverlay extends React.Component<LinkTypeOverlayProps> {
             target,
             title,
             href,
+            options,
         } = this.props;
 
+        if (!options || !options.targets) {
+            throw new Error('The ExternalLinkTypeOverlay needs atleast targets as options in order to work!');
+        }
+
+        const {
+            targets,
+        } = options;
```

Comment:
This is a bc break we should keep the component as it was and prefill the values:

```suggestion
        const {
            targets = {
                '_blank': 'sulu_admin.link_target_blank',
                '_self': 'sulu_admin.link_target_self',
                '_parent': 'sulu_admin.link_target_parent',
                '_top': 'sulu_admin.link_target_top',
            },
        } = options;
```